spec/changelog.yml

```diff
@@ -636,3 +636,6 @@
   - description: Introduce "value" property for constant_keyword fields
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/194
+  - description: Allow _ in github owner field
+    type: enhancement
+    link: https://github.com/elastic/package-spec/pull/668
```

Comment:
This change should be moved to the section for the next version, top of the file.

Reply:
Fixed.